server/graphql/v2/mutation/OrderMutations.js

```diff
@@ -494,15 +495,28 @@ const orderMutations = {
         include: [
           { association: 'paymentMethod' },
           { association: 'fromCollective', attributes: ['id', 'slug'] },
-          { association: 'collective', attributes: ['id', 'slug'] },
+          { association: 'collective', attributes: ['id', 'slug', 'HostCollectiveId'] },
         ],
       });
 
       // -- Some sanity checks to prevent issues --
       const paymentMethodIds = uniq(orders.map(order => order.PaymentMethodId).filter(Boolean));
       const ordersIds = orders.map(order => order.id);
+      let isAddedFunds;
+      let paymentMethod;
+
       for (const order of orders) {
-        const isUpdatingPaymentMethod = Boolean(fromAccount);
+        if (
+          fromAccount.HostCollectiveId !== order.collective.HostCollectiveId &&
+          fromAccount.type !== CollectiveType.USER
```

Comment:
Why ignore `USER`s here?

Reply:
@Betree : If the user selects any `USER` this condition; `fromAccount.HostCollectiveId !== order.collective.HostCollectiveId ` will always be true and we'll throw the error. Let me know what you think.

Reply:
Some hosts have the type "USER", and therefore have their `HostCollectiveId` set to themselves

Reply:
Suggested alternative:

```es6
if (fromAccount) {
  if (fromAccount.HostCollectiveId && fromAccount.HostCollectiveId !== order.collective.HostCollectiveId) {
    throw new ValidationFailed(`Orders cannot be moved to a different host`);
  }
}
```

Reply:
Ah, great point. I totally missed that hosts can be of type `USER`. 🤔 I've corrected as per your suggestion.